### PR TITLE
feat: update README with Secret Manager integration

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: cloudsql/mysql/database-sql/README.md [hunk 1]
```diff
@@ -177,11 +177,28 @@ gcloud run deploy run-sql --image gcr.io/[YOUR_PROJECT_ID]/run-sql \
...
+
+Create secrets via the command line:
+```sh
+echo -n "my-awesome-project:us-central1:my-cloud-sql-instance" | \
+    gcloud secrets versions add CLOUD_SQL_CONNECTION_NAME_SECRET --data-file=-
+```
+
+Deploy the service to Cloud Run specifying the env var name and secret name:
+```sh
+gcloud beta run deploy SERVICE --image gcr.io/[YOUR_PROJECT_ID]/run-sql \
```
Comment (enocom) [inline cloudsql/mysql/database-sql/README.md:195]: Why is this `beta`? At a quick glance, all these values are supported in the gcloud run command.

**Comment by enocom** [comment]
There's also a [Postgres README](https://github.com/GoogleCloudPlatform/golang-samples/tree/master/cloudsql/postgres/database-sql#deploy-to-cloud-run) that probably needs to be updated as well.

File: cloudsql/mysql/database-sql/README.md (hunk 1 above)
Comment (averikitsch) [reply cloudsql/mysql/database-sql/README.md:195]: the `--update-secrets` flag is in beta

Comment (enocom) [reply cloudsql/mysql/database-sql/README.md:195]: Oh. Got it. Thanks.

**Review by enocom — APPROVED** [review]

**Review by codyoss — APPROVED** [review]

**Pull request merged**